src/modules/menu/components/advancedTools/ImportFile/ImportFileOnlineTab.vue

```diff
@@ -85,10 +86,13 @@ async function loadFile() {
             timeout: REQUEST_TIMEOUT,
             responseType: 'arraybuffer',
         })
-        if (response.status !== 200) {
+        if (response instanceof GeoTIFF) {
+            await handleFileContent(store, response, fileUrl.value)
```

Comment:
See comment in `utils.js`. Since this is almost the same as on line 94 but calls it slightly differently, and yet `handleFileContent` does something entirely different when it's a GeoTIFF, I would argue we could have a separate function here (`handleGeoTiffContent` ?)